examples/clock.py

```diff
@@ -0,0 +1,74 @@
+import asyncio
+
+from yapapi.payload import vm
+from yapapi.services import Service
+
+from service_manager import ServiceManager
+
+
+class ProviderClock(Service):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.command_executed_cnt = 0
+
+    @classmethod
+    async def get_payload(cls):
+        #   We use a "standard" yapapi blender image (because every provider has it downloaded)
+        #   Could be any other working image, the only requirement is /bin/date command
+        image_hash = "9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae"
+        return await vm.repo(image_hash=image_hash)
+
+    async def run(self):
+        while True:
+            #   Q: If we want the datetime, Why not just use standard datetime library?
+            #   A: Because python is running on the **requestor** machine. Data is extracted from provider only
```

Comment:
```suggestion
            #   A: Because this python code is running on the **requestor** machine. The datetime is extracted from provider only
```

I had trouble to understand the explanation, happy to improve if it got it right.

Reply:
https://github.com/golemfactory/yapapi-service-manager/pull/6